compiler/modules/print.llvm/llvm.bal

```diff
@@ -36,14 +36,22 @@ public readonly class PointerValue {
 # Subtype of Value that refers to a constant
 public readonly class ConstValue {
     *Value;
-    string operand;
+    string|Unnamed operand;
```

Comment:
This makes ConstValue the same type as Value...  How can a ConstValue have referenced to Unnamed?

Reply:
It was there to allow casting between `ConstValue` and `PointerValue`. Fixed by adding explicit functions to create `ConstValue` from `Value` and `PointerValue`, and `PointerValue` from `ConstValue`.